dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java

```diff
@@ -1267,15 +1267,21 @@ protected Node methodInvocationConvert(Node node, TypeMirror formalType) {
     /**
      * Given a method element, its type at the call site, and a list of argument expressions, return
      * a list of {@link Node}s representing the arguments converted for a call of the method. This
-     * method applies to both method invocations and constructor calls.
+     * method applies to both method invocations and constructor calls. The constructor may need an
+     * extra enclosingExprType parameter when it is an inner class constructor with an explicit
+     * enclosing expression. The part of "Handle anonymous constructors" is moved from the helper
```

Comment:
Again, don't document what was or what happened, document what is the behavior now. Why would anyone care that some code was moved here? Document what the behavior is because that code was moved.

Reply:
Thanks! Done in my local